server/routes/api.js

```diff
@@ -219,6 +219,80 @@ router.delete('/event/:id', passport.authenticate('jwt', {session: false}), asyn
   }
 });
 
+// Add the user as an attendee to a specific event
+router.post('/attend/event', passport.authenticate('jwt', {session: false}), async (req, res) =>
+{
+    const eventID = req.body.eventID;
+    const userID = req.body.userID;
+    let succesful = true;
+
+    const member_event = new Member_Event(
+        {
+            date: new Date(),
+            paid: false,
+            member: userID,
+            event: eventID
+        }
+    );
+
+    member_event.save()    
+    .catch(err => 
+        {
+            console.log("Error while attending: " + err);
+            succesful = false;
+            return res.json({success: false});
+        }
+    );
+
+    if (succesful)
+    {
+        // After the attendance has been succesful, increase the amount of attendees for the event.
+        Event.findByIdAndUpdate({_id: eventID}, { $inc: {attendees: 1} })
+        .catch(err =>
+            {
+                if (err)
+                {
+                    console.log(err);
+                }
+            })
+
+        return res.json({success: true});
+    }
+});
+
+// Confirm if a user is attending an event or not
+router.get('/is/attending/:eventID/:userID', async (req, res) =>
+{
+    const eventID = req.params.eventID;
+    const userID = req.params.userID;
+    let attendees = [];
+    let found = false;
+
+    await Member_event.find({event: eventID})
```

Comment:
Hmm, I think the mongoose find() function allows searching by multiple fields, so this could maybe just be
`await Member_event.find({ event: eventID, member: memberID })`
Then it would simply be a case of if there's a hit, the member is attending the event, and if there isn't a hit, they're not :D

I could be wrong as well, but at least [what i found by googling](https://stackoverflow.com/questions/33627238/mongoose-find-with-multiple-conditions) makes it seem like that would be possible 😄

Reply:
Modified it to work like this and it is a WAY better way to do it :)